patches/core-sdk/0008-Add-target-rid-to-AppHostRuntimeIdentifiers.patch

```diff
@@ -0,0 +1,37 @@
+From 62020f2c4fdf2c0389b4106cac9286f1c3a9eb7c Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Thu, 16 Jan 2020 08:37:34 +0100
+Subject: [PATCH] Add target rid to AppHostRuntimeIdentifiers
+
+---
+ src/redist/targets/GenerateBundledVersions.targets | 7 ++++++-
+ 1 file changed, 6 insertions(+), 1 deletion(-)
+
+diff --git a/src/redist/targets/GenerateBundledVersions.targets b/src/redist/targets/GenerateBundledVersions.targets
+index a214842a0..2c350f15f 100644
+--- a/src/redist/targets/GenerateBundledVersions.targets
++++ b/src/redist/targets/GenerateBundledVersions.targets
+@@ -48,6 +48,11 @@
+ 
+       <NetCoreRuntimePackRids Include="@(NetCore30RuntimePackRids)"/>
+ 
++      <NetCoreAppHostPackRids Include="
++          $(ProductMonikerRid);
++          @(NetCoreRuntimePackRids)
++          " />
++
+       <AspNetCore30RuntimePackRids Include="
+         win-x64;
+         win-x86;
+@@ -168,7 +173,7 @@ Copyright (c) .NET Foundation. All rights reserved.
+                       TargetFramework="netcoreapp3.1"
+                       AppHostPackNamePattern="Microsoft.NETCore.App.Host.**RID**"
+                       AppHostPackVersion="$(_NETCoreAppPackageVersion)"
+-                      AppHostRuntimeIdentifiers="@(NetCoreRuntimePackRids, '%3B')"
++                      AppHostRuntimeIdentifiers="@(NetCoreAppHostPackRids, '%3B')"
```

Comment:
This is the change for 3.1.

It's possible there is an apphost also for 3.0 (from installing the 3.0 sdk), but we can't add it to the 3.0 `AppHostRuntimeIdentifiers` because that would cause a failure in case it isn't installed.
We could make the sdk look into the packs folder so it can figure out itself what apphosts are available locally. Then we don't have to add them explicitly here either.